changelogs/current.yaml

```diff
@@ -8,6 +8,9 @@ minor_behavior_changes:
 - area: adaptive concurrency filter stats
   change: |
     Multiply the gradient value stat by 1000 to make it more granular (values will range between 500 and 2000).
+- area: stateful_session
+  change: |
+    Support 0 TTL for proto-encoded cookies.
```

Comment:
Please add what is the expected behavior for 0 TTL, and consider moving this under `bug_fixes` in the same file below.

Reply:
Done.